cpp/plugins/cucim.kit.cumed/tests/test_basic.cpp

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright (c) 2021, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "config.h"
+
+#include <catch2/catch.hpp>
+#include <chrono>
+
+TEST_CASE("Verify file", "[test_basic.cpp]")
+{
+    REQUIRE(1 == 1);
```

Comment:
Were we wanting to add more here?

Reply:
Currently, it is just a placeholder to implement unit tests for cumed plugin.